MAINTAINERS.md

```diff
@@ -1,30 +1,18 @@
 # Maintainers
 
 See [Governance](Documentation/community/governance/commit_access.rst) for
-governance, commit, and vote guidelines as well as maintainer responsibilities.
+governance, commit, and vote guidelines as well as committer responsibilities.
 Everybody listed is a committer as per governance definition.
 
-All committers and maintainers have identical voting rights. Maintainers have
-additional administrative and janitorial responsibilities.
-
-Regardless of maintainer or committer status, any code contribution is subject
+Regardless of committer status, any code contribution is subject
 to code review requirements as per [CODEOWNERS](CODEOWNERS) and PRs get merged
-by the "janitor role" which is rotated between all committers.
-
-## Cilium Maintainers
+by the "tophat role" which is rotated between all committers.
```

Comment:
```suggestion
by the "tophat role" which is rotated between all committers as well as by any committer is the tests were successful and the reviews from all codeowners were accepted.
```

Reply:
Thinking about this again, should we just end this sentence after CODEOWNERS since who merges could change again in the future?

Reply:
I agree with Bill, there's no need to go into details about the mechanics of merging PRs in `MAINTAINERS.md`.

Reply:
I even wonder if this paragraph around "Regardless of ..." should just be dropped entirely. That was there to make it clear that Maintainers are not elevated above the standards required for all Committers. The governance doc already clarifies this.

If we think that people will come to the manitainers doc to look for information about how to get PRs merged, then maybe a simple link to https://docs.cilium.io/en/stable/contributing/development/contributing_guide would suffice. But I'm also OK to just omit that.

Reply:
I'm going to delete it